omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-centos7-ice3.6.rst

```diff
@@ -1,26 +1,31 @@
-OMERO.web walkthrough installation CentOS 7 and IcePy 3.6
-=========================================================
+OMERO.web deployed separately from OMERO.server on CentOS 7 and IcePy 3.6
+=========================================================================
 
+Please first read :doc:`../../server-centos7-ice36`.
 
-For convenience in this walkthrough the main OMERO.web configuration options have been defined as environment variables. When following this walkthrough you can either use your own values, or alternatively source the following file::
+
+This is an example walkthrough for installing OMERO.web decoupled from the OMERO.server in a **virtual environment** using OMERO.py and a dedicated system user. Installing OMERO.web in a virtual environment is the preferred way. For convenience in this walkthrough the main OMERO.web configuration options have been defined as environment variables. When following this walkthrough you can either use your own values, or alternatively use the following ones::
```

Comment:
wrap this long line

Reply:
the way the template introduced  initially does not work very well
everything ends up being indented

Reply:
Ouch, okay. Maybe something to work on in the longer term then.

Reply:
Will look at the template post release I think